CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 
 ### Enhancements
 
-- None
+- `prefect make-user-config` now creates default user config from src/prefect/config.toml
```

Comment:
```suggestion
- `prefect make-user-config` now creates default user config from src/prefect/config.toml - [#904](https://github.com/PrefectHQ/prefect/issues/904)
```
we typically link the change to the issue it closes; if no issue exists, we link directly to the PR